airflow/io/store/path.py

```diff
@@ -174,6 +174,10 @@ def __truediv__(self, other) -> ObjectStoragePath:
         path = f"{self_path.rstrip(self.sep)}/{other_path.lstrip(self.sep)}"
         return ObjectStoragePath(path, conn_id=self._conn_id)
 
+    def strip_protocol(self) -> str:
```

Comment:
should this be an internal method too?

Reply:
I should kick this out. It's a left over.